common-lib/src/cmr/common/util.clj

```diff
@@ -155,6 +155,10 @@
                           {:curr-val (first next-vals)
                            :next-vals (rest next-vals)}))))))
 
+(def seconds->milliseconds
```

Comment:
The -> in the name makes me think it does the conversion rather than being a constant. Could we either make it into a converter or make the name a bit more clear that it's the number of milliseconds in a second?

Reply:
I can rename

Reply:
seconds-in-milliseconds